.mailmap

```diff
@@ -14,163 +18,203 @@
+Ginés Carrascal <gines_carrascal@es.ibm.com> <gines.carrascal@gmail.com>
 Hiroshi Horii <horii@jp.ibm.com> <hhorii@users.noreply.github.com>
-Ian Gould <iangould@IANs-MacBook.local>
 Igor Olegovich Sokolov <iso@zurich.ibm.com> <ISO@zurich.ibm.com>
+Ikko Hamamura <ikkoham@users.noreply.github.com>
+Isha Rajput <33170219+irajput@users.noreply.github.com>
 Ismael Faro Sertage <ismael.faro1@ibm.com> <ismael.faro.sertage@gmail.com>
 Ismael Faro Sertage <ismael.faro1@ibm.com> <ismael.faro1@ibm.com>
 Ismail Yunus Akhalwaya <ismaila@za.ibm.com>
 Ismail Yunus Akhalwaya <ismaila@za.ibm.com> <30803146+ismaila-at-za-ibm@users.noreply.github.com>
-Jack J. Woehr <4604036+jwoehr@users.noreply.github.com>
+Ivan Carvalho <ivancarv@student.ubc.ca> <ivan.ivancps.cn@gmail.com>
+Jack J. Woehr <4604036+jwoehr@users.noreply.github.com> <jwoehr@softwoehr.com>
+Jake Lishman <jake.lishman@ibm.com>
 Jake Lishman <jake.lishman@ibm.com> <jake@binhbar.com>
 James R. Garrison <garrison@ibm.com>
 James R. Garrison <garrison@ibm.com> <jim@garrison.cc>
 James Seaward <58116376+JamesSeaward@users.noreply.github.com>
-Jan Müggenburg <jan.mueggenburg@ibm.com>
-Jan Müggenburg <jan.mueggenburg@ibm.com> <jan@oc6058543760.ibm.com>
 Jay M. Gambetta <jay.gambetta@us.ibm.com>
 Jerry M. Chow <chowmj@us.ibm.com>
 Jessica Wilson <jessicalostinspace@gmail.com>
 Jesús Pérez <jesusper@us.ibm.com> <jesusprubio@gmail.com>
+Jesús Sistos <46694558+jsistos@users.noreply.github.com> <jesussistos.josb@gmail.com>
 Johannes Weidenfeller <54772951+Knabberjoe@users.noreply.github.com>
 Jonathan A. Wildstrom <34069155+jwildstr-ibm@users.noreply.github.com>
 Jonathan A. Wildstrom <34069155+jwildstr-ibm@users.noreply.github.com> <jwildstr@us.ibm.com>
 Jonathan Shoemaker <Jonathan-Shoemaker@ibm.com>
+Jonathan Shoemaker <Jonathan-Shoemaker@ibm.com> <66969627+jonathan-shoemaker@users.noreply.github.com>
 Jordan Connor <jordanhconnor@gmail.com>
 Jorge Carballo <carballo@us.ibm.com> <jelcaf@gmail.com>
 Juan Cruz-Benito <juan.cruz@ibm.com>
 Juan Gomez-Mosquera <atilag@gmail.com>
 Juan Gomez-Mosquera <atilag@gmail.com> <Juan.Gomez1@ibm.com>
+Juan Luis Sánchez Toural <toural@es.ibm.com>
 Julien Gacon <jul@zurich.ibm.com>
+Julien Gacon <jul@zurich.ibm.com> <gaconju@gmail.com>
 Julien Gacon <jul@zurich.ibm.com> <jules.gacon@googlemail.com>
+Kazumasa Umezawa <e35372@jp.ibm.com>
+Kevin J. Sung <kevinsung@users.noreply.github.com> <kevinjefferysung@gmail.com>
+Kevin J. Sung <kevinsung@users.noreply.github.com> <kevjsung@umich.edu>
 Kevin Krsulich <kevin.krsulich@ibm.com> <kevin@krsulich.net>
+Lauren Capelluto <lcapelluto@users.noreply.github.com> <laurencapelluto@gmail.com>
 Lev S. Bishop <18673315+levbishop@users.noreply.github.com>
+Luciano Bello <luciano.bello@ibm.com> <bel@zurich.ibm.com>
 Luciano Bello <luciano.bello@ibm.com> <lbello@gmail.com>
 Luciano Bello <luciano.bello@ibm.com> <luciano.bello@ibm.com>
 Luciano Bello <luciano.bello@ibm.com> <luciano@debian.org>
```

Comment:
These maybe all need flipping to Luciano's current email address - `bel@zurich.ibm.com`.

Reply:
I see what you mean, but again treating the mailmap as a way to build a database of "author preferred name-or-nym <email-as-uuid>" then I don't think theres a need to, and if I change it here then I should probably change it for all the qiskit project mailmaps (Luciano has been active in 18 qiskit org repos) if we want records to be comparable across repos